zio-http/src/test/scala/zhttp/internal/HttpRunnableSpec.scala

```diff
@@ -36,7 +36,11 @@ abstract class HttpRunnableSpec(port: Int) extends DefaultRunnableSpec { self =>
   def serve[R <: Has[_]](
     app: HttpApp[R, Throwable],
   ): ZManaged[R with EventLoopGroup with ServerChannelFactory, Nothing, Unit] =
-    Server.make(Server.app(app) ++ Server.port(port) ++ Server.paranoidLeakDetection).orDie
+    Server
+      .make(
+        Server.app(app) ++ Server.port(port) ++ Server.error(err => UIO(println(err))) ++ Server.paranoidLeakDetection,
```

Comment:
This change won't be required if the previous comment is fixed.

Reply:
done